tests/unittests/core/test_branch_config.py

```diff
@@ -407,22 +408,12 @@ def test_reset_dimension(self, parent_config, new_config):
         assert isinstance(conflict, NewDimensionConflict)
         assert len(conflicts.get_resolved()) == 0
 
+    @pytest.mark.skip(reason="Reinstate once --version has been added.")
     def test_name_experiment(self, parent_config, child_config, create_db_instance):
-        """Test if changing the experiment names work for valid name"""
+        """Test if having the same experiment name does not create a conflict."""
```

Comment:
This should be turned into a case where we branch from parent which has a child, and then the test stays the same, changing the name we resolve the conflict.